packages/components-js/projects/components-manager-cli/library-entry/with-prefix.spec.ts

```diff
@@ -0,0 +1,109 @@
+import * as fromComponentsManagerCore from '@porsche-design-system/components-manager-core';
+import { load } from './with-prefix';
+import type { EntryConfig } from '../shared-definitions/entry-config';
+
+declare global {
+  var CM_CONFIG: EntryConfig;
+}
+
+global.CM_CONFIG = {
+  version: '1.0.0',
+  script: 'some.js',
+  tempEntryPointFilePath: 'another.js',
+};
+
+describe('load()', () => {
+  beforeEach(() => {
+    // @ts-ignore
+    delete window.PORSCHE_DESIGN_SYSTEM_CDN;
+    delete document.porscheDesignSystem;
+  });
+
+  it('should call loadComponentLibrary() with correct default parameters', () => {
+    const spy = jest.spyOn(fromComponentsManagerCore, 'loadComponentLibrary');
+
+    load();
+    expect(spy).toBeCalledWith({ prefix: '', ...global.CM_CONFIG });
+  });
+
+  it('should call loadComponentLibrary() with correct prefix parameters', () => {
+    const spy = jest.spyOn(fromComponentsManagerCore, 'loadComponentLibrary');
+
+    load({ prefix: 'my-prefix' });
+    expect(spy).toBeCalledWith({ prefix: 'my-prefix', ...global.CM_CONFIG });
+  });
+
+  it('should by default set window.PORSCHE_DESIGN_SYSTEM_CDN="auto"', () => {
+    load();
+    expect(window.PORSCHE_DESIGN_SYSTEM_CDN).toBe('auto');
+  });
+
+  it('should for { cdn: "auto" } set window.PORSCHE_DESIGN_SYSTEM_CDN="auto"', () => {
```

Comment:
```suggestion
  it('should for { cdn: "auto" } set window.PORSCHE_DESIGN_SYSTEM_CDN="auto" for backwards compatiblity', () => {
```
Maybe to make things clearer?